docs/sources/enterprise/usage-insights/improved-search.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Improved dashboard search"
```

Comment:
The title and description sound like a release notes item rather than core documentation. An alternative title is, "Sort dashboards using insights data".

Reply:
+1

Reply:
I made the update and tried to improve the documentation.